### DISCUSSION TITLE
Welcome to swagger-stats Discussions!

### DISCUSSION THREAD

**Discussion by sv2** [slanatech/swagger-stats#125] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by bhupesh-sf** [comment]
Hi,

This is Bhupesh Gupta a Senior Technical Architect.

We are using this module for the awesome stats and metrics it gives. Recently we planned to shift everything to Elastic and get away from Prometheus, Grafana, etc. We were already using elastic for logs. I implemented tracing using opentelemetry and used elastic exporter for the same. And now I can see my tracing also in elastic APM. But the next thing was to have metrics also in elastic and I know swagger-stats gives an option for elastic. 

But now a couple of questions here:
1. Do we get everything on elastic? I can see request and response are going in elastic but cannot see system metrics like CPU, memory, etc.
2. Is there a way I can add trace ID with the metrics so I can see everything logs, traces and metrics for a single trace ID.

Any help here will be appreciated.
Thanks.

**Comment by dsgrangs-paul** [comment]
Hi
 My use case is the following : 
  - Using swagger-stats module in ApolloServer performing calls to REST API through Swagger framework, and swagger using axios module to perform http calls
  - But I cannot see any detection of RESTAPI calls being done by swagger-stats : when accessing   http://localhost:4000/swagger-stats/#/login  I can see the logs of ' onAuthenticate' but console does not show any RESTAPI calls 
   - I suppose the match between the call being done on REST API 
           for example "return client.apis.PCloudCloudConnections.pcloud_cloudconnections_getall(getBody);" 
      and the REST API being defined in swagger.json 
           same example : 
                "/pcloud/v1/cloud-instances/{cloud_instance_id}/cloud-connections": {
                "get": {
                "tags": [
                    "PCloudCloudConnections"
                ],
                "operationId": "pcloud_cloudconnections_getall",
        is not propermy done
        And I wonder how the match is done ??
        
 I 'm interested in any clue.
 At the end, the call to REST API is transformed into an "axios" module request by my execution REST API engine
 
 How swagger-stats could match the real call being done with the definition of the call in swagger.json ?
   
 Thank you
